source/frontend/src/models/generate/GenerateFile.ts

```diff
@@ -7,19 +7,45 @@ import { GenerateProperty } from './GenerateProperty';
 export class Api_GenerateFile {
```

Comment:
Is the Api_GenerateFile class a generic data generation class?

Reply:
its a mapping between our api model and a model used for generation.
The idea was to use a similar pattern for how to map between our api models and frontend.